dragonfly_doe2/doe/material.py

```diff
@@ -51,16 +53,15 @@ class MassMaterial:
     density: float
     specific_heat: float
 
-
     @classmethod
     def from_hb_material(cls, material: EnergyMaterial):
-        name = material.display_name
-        thickness = _unit_convertor([material.thickness], 'ft', 'm')
-        conductivity = _unit_convertor([material.conductivity], 'Btu/h-ft2', 'W/m2')
-        density = material.density / 16.018
-        specific_heat = _unit_convertor([material.specific_heat], 'Btu/lb', 'J/kg')
+        name = short_name(material.display_name, 32)
+        thickness = _unit_convertor([material.thickness, 3], 'ft', 'm')
+        conductivity = _unit_convertor([material.conductivity, 3], 'Btu/h-ft2', 'W/m2')
+        density = round(material.density / 16.018, 3)
+        specific_heat = _unit_convertor([material.specific_heat, 3], 'Btu/lb', 'J/kg')
```

Comment:
You don't need to pass 3 here, right? It is hard-coded in the function.

```suggestion
        name = short_name(material.display_name, 32)
        thickness = _unit_convertor([material.thickness], 'ft', 'm')
        conductivity = _unit_convertor([material.conductivity], 'Btu/h-ft2', 'W/m2')
        density = round(material.density / 16.018, 3)
        specific_heat = _unit_convertor([material.specific_heat], 'Btu/lb', 'J/kg')
```